README.md

```diff
@@ -92,12 +92,6 @@ This library makes localizing WPF applications at runtime using resx files a bre
 
 > Licensed under [The Code Project Open License](http://www.codeproject.com/info/cpol10.aspx).
 
-###[EventHook](https://github.com/justcoding121/Windows-User-Action-Hook)
-
-> A one stop library for global windows user actions such mouse, keyboard, clipboard, website visit & print events.
-
-This library allows Rubberduck to detect righ-click actions in the active code pane, to dynamically enable/disable menu commands depending on the current context/selection. We're also using it to capture keypresses, to trigger a reparse of the current module as it's being modified.
-
```

Comment:
It saddens me to remove this library - the API was very promising.. but I couldn't get it to work along with the hotkey hook, and couldn't get a PR done either.